NEWS.md

```diff
@@ -2,6 +2,11 @@
 
 ## Polars R Package (development version)
 
+### New features
+
+- `$drop()` now accepts several character vectors, such as `$drop("a", "b")` 
```

Comment:
I suspect this is a potentially breaking change.
In other words, if you specify it with the columns argument instead of the position argument, won't unexpected behavior occur?

Reply:
Good catch. Actually I think there's a bug in `dots_to_colnames()` because passing `c()` in second values doesn't work:

```r
### this works
pl$DataFrame(mtcars)$drop(c("mpg", "drat"), "hp") |> ncol()
[1] 8

### this doesn't
pl$DataFrame(mtcars)$drop("hp", c("mpg", "drat")) |> ncol()

Error: Execution halted with the following contexts
   0: In R: in $drop():
   0: During function call [ncol(pl$DataFrame(mtcars)$drop("hp", c("mpg", "drat")))]
   1: When constructing a Column Expr
   2: The argument [ `...` ] caused an error
   3: Possibly because element no. [1] 
   4: Expected a value of type [alloc::string::String]
   5: Got value [Rvalue: ["mpg", "drat"], Rsexp: Strings, Rclass: ["character"]]
```

Reply:
This is a problem of `pl$col()`, not `dots_to_colnames()`.

```r
> library(polars)

> pl$col(c("mpg", "drat"), "hp")
polars Expr: cols(["mpg", "drat", "hp"])

> pl$col("hp", c("mpg", "drat"))
Error: Execution halted with the following contexts
   0: In R: in pl$col()
   0: During function call [pl$col("hp", c("mpg", "drat"))]
   1: When constructing a Column Expr
   2: The argument [ `...` ] caused an error
   3: Possibly because element no. [1] 
   4: Expected a value of type [alloc::string::String]
   5: Got value [Rvalue: ["mpg", "drat"], Rsexp: Strings, Rclass: ["character"]]

> pl$col("hp", name = c("mpg", "drat"))
polars Expr: cols(["mpg", "drat", "hp"])

> pl$col("hp", columns = c("mpg", "drat"))
Error: Execution halted with the following contexts
   0: In R: in pl$col()
   0: During function call [pl$col("hp", columns = c("mpg", "drat"))]
   1: When constructing a Column Expr
   2: The argument [ `...` ] caused an error
   3: Possibly because element no. [1] 
   4: Expected a value of type [alloc::string::String]
   5: Got value [Rvalue: ["mpg", "drat"], Rsexp: Strings, Rclass: ["character"]]
```

We should rewrite `pl$col()`...(#912)

Reply:
Something like `pl$DataFrame(mtcars)$drop(columns = c("mpg", "drat"))` works in the main branch, but should fail in this PR because of `pl$col(name = "", ...)`.

Reply:
Indeed, I've updated news, thanks